src/org/authInfo.ts

```diff
@@ -756,10 +759,9 @@ export class AuthInfo extends AsyncOptionalCreatable<AuthInfo.Options> {
     }
   }
 
-  private getInstanceUrl(options: unknown, aggregator: ConfigAggregator) {
+  private getInstanceUrl(options: AuthOptions, aggregator: ConfigAggregator) {
     const instanceUrl =
-      getString(options, 'instanceUrl') ||
-      (aggregator.getPropertyValue(OrgConfigProperties.ORG_INSTANCE_URL) as string);
+      options?.instanceUrl || (aggregator.getPropertyValue(OrgConfigProperties.ORG_INSTANCE_URL) as string);
```

Comment:
side comment, not for this PR

wouldn't it be nice if getPropertyValue knew the types of at least the known properties?